### PR TITLE
Onyx & Diamond Sleet APT AExploitation Activity

### PR DESCRIPTION
**Pull request by nasbench**
### Summary of the Pull Request

This PR adds new ET rules related to Diamond & Onyx Sleet activity as described in MS [report](https://www.microsoft.com/en-us/security/blog/2023/10/18/multiple-north-korean-threat-actors-exploiting-the-teamcity-cve-2023-42793-vulnerability/)

### Changelog

fix: Potential System DLL Sideloading From Non System Locations
new: Diamond Sleet APT DNS Communication Indicators
new: Diamond Sleet APT File Creation Indicators
new: Diamond Sleet APT DLL Sideloading Indicators
new: Diamond Sleet APT Process Activity Indicators
new: Diamond Sleet APT Scheduled Task Creation - Registry
new: Diamond Sleet APT Scheduled Task Creation
new: Onyx Sleet APT File Creation Indicators

### Example Log Event

N/A

### Fixed Issues

N/A

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

File: rules-emerging-threats/2023/TA/Diamond-Sleet/proc_creation_win_apt_diamond_sleet_indicators.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+date: 2023/10/24
+tags:
+    - attack.execution
+    - detection.emerging_threats
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        CommandLine|contains: ' uTYNkfKxHiZrx3KJ'
```
Comment (frack113) [inline rules-emerging-threats/2023/TA/Diamond-Sleet/proc_creation_win_apt_diamond_sleet_indicators.yml:17]:
```suggestion
        CommandLine|endswith: ' uTYNkfKxHiZrx3KJ'
```

File: rules-emerging-threats/2023/TA/Diamond-Sleet/proc_creation_win_apt_diamond_sleet_indicators.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+title: Diamond Sleet APT Process Activity Indicators
+id: b5495d8d-24ad-4a44-8caf-ceae9a07a5c2
+status: experimental
+description: Detects process creation activity indicators related to Diamond Sleet APT
+references:
+    - https://www.microsoft.com/en-us/security/blog/2023/10/18/multiple-north-korean-threat-actors-exploiting-the-teamcity-cve-2023-42793-vulnerability/
```
Comment (frack113) [inline rules-emerging-threats/2023/TA/Diamond-Sleet/proc_creation_win_apt_diamond_sleet_indicators.yml:6]:
```suggestion
    - https://www.microsoft.com/en-us/security/blog/2023/10/18/multiple-north-korean-threat-actors-exploiting-the-teamcity-cve-2023-42793-vulnerability/
    - https://securelist.com/the-lazarus-group-deathnote-campaign/109490/
```
Same argument  `Command line: %APPDATA%\USOShared\CameraSettingsUIHost.exe uTYNkfKxHiZrx3KJ`

File: rules-emerging-threats/2023/TA/Diamond-Sleet/proc_creation_win_apt_diamond_sleet_indicators.yml (hunk 1 above)
Comment (nasbench) [reply rules-emerging-threats/2023/TA/Diamond-Sleet/proc_creation_win_apt_diamond_sleet_indicators.yml:17]: Its unique enough that we don't need this endswith

File: rules-emerging-threats/2023/TA/Diamond-Sleet/proc_creation_win_apt_diamond_sleet_indicators.yml (hunk 2 above)
Comment (nasbench) [reply rules-emerging-threats/2023/TA/Diamond-Sleet/proc_creation_win_apt_diamond_sleet_indicators.yml:6]: Unrelated to the emerging threat as its reported by a diff vendor. Only related in CLI

**Review by frack113 — COMMENTED** [review]
Some possible change

**Review by frack113 — APPROVED** [review]

**Pull request merged**